deployments/warp_routes/USDC/eclipse/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "USDC",
```

Comment:
copying my comment from the monorepo PR:

I'd probably do `USD Coin` to match what is on [Ethereum](https://etherscan.io/token/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48#readProxyContract#F16) and [Solana](https://solscan.io/token/EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v#metadata)